### PR TITLE
57689

### PR DESCRIPTION
**Pull request by robinwpdeveloper**
Trac ticket: https://core.trac.wordpress.org/ticket/57689

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by robinwpdeveloper** [comment]
Hi @desrosj 
https://github.com/robinwpdeveloper/wordpress-develop/tree/57689 is now up to date with the latest trunk.

I think this is what you meant by refreshing PR.

**Comment by audrasjb** [comment]
Hi @robinwpdeveloper, more than just refreshing against trunk, the idea was to implement the feedback of comment:8 (see the Trac ticket)

File: src/wp-content/themes/twentytwenty/readme.txt [hunk 1]
```diff
@@ -22,6 +22,11 @@ you pick, ensuring a high, accessible color contrast for your visitors.
 
 == Changelog ==
 
+= 2.2 =
+* Released: March 28, 2023
+
+https://wordpress.org/support/article/twenty-twenty-changelog/#Version_2.2
```
Comment (costdev) [inline src/wp-content/themes/twentytwenty/readme.txt]:
```suggestion
https://wordpress.org/documentation/article/twenty-twenty-changelog/#Version_2.2
```
Per [this comment](https://core.trac.wordpress.org/ticket/57689#comment:9).

File: src/wp-content/themes/twentytwentyone/readme.txt [hunk 2]
```diff
@@ -30,6 +30,11 @@ No data is saved in the database or transferred.
 
 == Changelog ==
 
+= 1.8 =
+* Released: March 28, 2023
+
+https://wordpress.org/support/article/twenty-twenty-one-changelog#Version_1.8
```
Comment (costdev) [inline src/wp-content/themes/twentytwentyone/readme.txt]:
```suggestion
https://wordpress.org/documentation/article/twenty-twenty-one-changelog/#Version_1.8
```
Per [this comment](https://core.trac.wordpress.org/ticket/57689#comment:9).

File: src/wp-content/themes/twentytwentythree/readme.txt [hunk 3]
```diff
@@ -15,6 +15,11 @@ Whether you want to build a complex or incredibly simple website, you can do it
 
 == Changelog ==
 
+= 1.1 =
+* Released: March 28, 2023
+
+https://wordpress.org/support/article/twenty-twenty-three-changelog#Version_1.1
```
Comment (costdev) [inline src/wp-content/themes/twentytwentythree/readme.txt]:
```suggestion
https://wordpress.org/documentation/article/twenty-twenty-three-changelog/#Version_1.1
```
Per [this comment](https://core.trac.wordpress.org/ticket/57689#comment:9).

File: src/wp-content/themes/twentytwentytwo/readme.txt [hunk 4]
```diff
@@ -17,6 +17,11 @@ Whether you’re building a single-page website, a blog, a business website, or
 
 == Changelog ==
 
+= 1.4 =
+* Released: March 28, 2023
+
+https://wordpress.org/support/article/twenty-twenty-two-changelog#Version_1.4
```
Comment (costdev) [inline src/wp-content/themes/twentytwentytwo/readme.txt]:
```suggestion
https://wordpress.org/documentation/article/twenty-twenty-two-changelog/#Version_1.4
```
Per [this comment](https://core.trac.wordpress.org/ticket/57689#comment:9).

**Review by costdev — CHANGES_REQUESTED** [review]
Thanks for the PR @robinwpdeveloper! All the dates look good to me, pending confirmation about the three skipped files that need some feedback.

I've left some requested changes per [this comment on the ticket](https://core.trac.wordpress.org/ticket/57689#comment:9)

**Comment by robinwpdeveloper** [comment]
Thanks @audrasjb  and @costdev.
Requested changes are made.

**Review by desrosj — APPROVED** [review]
Thanks all! I've pushed a few more date/version changes that I noticed reviewing, and I also fixed the missing changes causing two themes to have unversioned changes after running `build` scripts.

**Comment by desrosj** [comment]
Merged in https://core.trac.wordpress.org/changeset/55476

**Pull request closed without merging**